features/visitor_can_create_new_task.feature

```diff
@@ -0,0 +1,27 @@
+Feature: A visitor can create new task
+    As a visitor,
+    In order to get my furniture refurbished
+    I would to be able create a new task
+
+    Scenario: Create task [Happy path]
+        When I am at the index page
+        And I click on 'New Task' link
+        Then I should be on New Task page
```

Comment:
No need to have this assertion in the middle of the scenario. Not a blocker though.